site/docs/v1/tech/installation-guide/cloud.adoc

```diff
@@ -256,6 +257,29 @@ image::installation-guides/cloud/deployments-destroying.png[The deployment is be
 
 After the deployment is completely removed, it will have a "Destroyed" state on the [breadcrumb]#Deployments# tab. At this point you will no longer be charged for this deployment.
 
+=== Migrating a Deployment
+
+You can migrate your user data to a FusionAuth Cloud deployment or from FusionAuth Cloud to a self-hosted FusionAuth instance. 
+
+==== Migrating To FusionAuth Cloud
+
+To migrate from a self-hosted instance to FusionAuth Cloud:
+
+* You must be running PostgreSQL. If you are running MySQL, please migrate to a PostgreSQL database locally. Here's a link:https://fusionauth.io/community/forum/topic/985/migrating-from-mysql-to-postgresql[forum post on how to do this].
+* Export your FusionAuth database to a file.
+* Encrypt the export file with a symmetric solution such as gpg.
+* Purchase a FusionAuth Cloud deployment that meets your needs. Select the version of FusionAuth that is the same as your local installation.
+* link:https://account.fusionauth.io/account/support/[Open a support ticket] and provide the database export file, the version of PostgreSQL you are running, and the name of the new FusionAuth deployment.
+* Provide the password for the database export through another channel. You can mention the channel you'd prefer in the support ticket. Slack, email and a separate support ticket are all options.
+
+Within 1-2 business days, we will upload your data to your FusionAuth Cloud deployment.
+
+==== Migrating Away From FusionAuth Cloud
+
+To migrate from FusionAuth Cloud to a self-hosted instance, please request a database export, as documented in <<Accessing User Data>>. After you have the data, install it in a PostgreSQL database and configure your self-hosted instance to point to that database. Make sure that the self-hosted FusionAuth instance is the same version as your Cloud instance was.
```

Comment:
Are customers able to see the version of PG that we are running in the cloud?  They will need a way to confirm version compatibility as well.

Reply:
Good point. I mentioned they should open a support ticket. Filed an issue as well: https://github.com/FusionAuth/fusionauth-issues/issues/1390